src/librbd/internal.cc

```diff
@@ -1205,7 +1205,16 @@ int mirror_image_disable_internal(ImageCtx *ictx, bool force,
   {
     CephContext *cct = (CephContext *)io_ctx.cct();
     bool old_format = cct->_conf->rbd_default_format == 1;
-    uint64_t features = old_format ? 0 : cct->_conf->rbd_default_features;
+    uint64_t features;
+    if (old_format)
+      features = 0;
+    else {
+      int r = parse_rbd_default_features(cct->_conf->rbd_default_features, features);
+      if (r < 0) {
+        CephContext *cct = (CephContext *)io_ctx.cct();
+        ldout(cct, 1) << "Warning: error in rbd_default_features config value" << dendl;
```

Comment:
Might be nice to move the logging to within the parse method (just pass the CephContext) so that you can log which values caused an error.